src/Routes/RosSystemDetail/ros-details-page.scss

```diff
@@ -1,18 +1,29 @@
 .ros-system-info {
-  .pf-c-description-list.expanded-row {    
+  .pf-c-description-list.expanded-row {
+    font-size: var(--pf-global--FontSize--md);
+
     dt {
       font-weight: var(--pf-global--FontWeight--normal);
       
       &:after {
         content: ":";
       }
     }
+    row-gap: 0;
   }
   .pf-c-description-list__group {
     row-gap: 0;
+
+    .pf-c-description-list__term .pf-c-description-list__text {
+      font-size: var(--pf-global--FontSize--md);
+    }
   }
 }
 
+.ros-system-info > .pf-l-flex.pf-m-column {
+  margin-bottom: var(--pf-l-flex--spacer);
+}
+
 .detail-header-container {
```

Comment:
@kgaikwad as we updated the layout to stack, can you please remove this from the file as we don't need it anymore?